web/src/routes/LandingPage.tsx

```diff
@@ -23,13 +22,13 @@ const LandingPage = ({ cities, match }: PropsType): ReactElement => {
   const pageTitle = t('pageTitle')
   const metaDescription = t('metaDescription', { appName: buildConfig().appName })
 
-  useScrollToTopOnMount()
```

Comment:
Why did you remove this?

Reply:
Its no longer needed on Landing page, as navigating back with the browser automatically jumps to the top.